### PR TITLE
Feature branch

### PR DESCRIPTION
**Pull request by Oitur**
# Microverse solo project #2 (Using Bootstrap)
Using Bootstrap framework to build a [Newsweek](https://www.newsweek.com/) clone

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,157 @@
+nav {
+  background-color: red;
+  max-width: 1310px;
+  margin: 0 auto;
```
Comment (bolah2009) [inline css/style.css]: Consider using a `Bootstrap` class name for this selector element and in all other occurrences instead.

`mx-auto` Bootstrap class name can be used instead. See [horizontal centering](https://getbootstrap.com/docs/4.1/utilities/spacing/#horizontal-centering) in bootstrap documentation.

#### Please, lets do this for all other occurrences. :blush:

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+}
+
+main {
+  max-width: 1310px;
+  margin: 0 auto;
+  padding: 0 15px;
+}
+
+.navbar {
+  padding: 0;
```
Comment (bolah2009) [inline css/style.css]: Consider using a `Bootstrap` class name for this selector element and in all other occurrences instead.

`p-0` Bootstrap class name can be used instead. See [Spacing](https://getbootstrap.com/docs/4.1/utilities/spacing) in bootstrap documentation.

#### Please, lets do this for all other occurrences. :blush:

File: css/style.css [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+  min-width: 464px;
+}
+
+.container-fluid {
+  padding: 0;
+}
+
+.widest {
+  width: 100%;
```
Comment (bolah2009) [inline css/style.css]: Consider using a `Bootstrap` class name for this selector element and in all other occurrences instead.

`w-100` Bootstrap class name can be used instead. See [Sizing](https://getbootstrap.com/docs/4.1/utilities/sizing/) in bootstrap documentation.

#### Please, lets do this for all other occurrences. :blush:

File: css/style.css [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+  margin: 10px 0;
+}
+
+#nw-logo {
+  height: 50px;
+  padding-top: 5px;
+}
+
+.today {
+  display: none;
```
Comment (bolah2009) [inline css/style.css]: Consider using a `Bootstrap` class name for this selector element and in all other occurrences instead.

`d-none` Bootstrap class name can be used instead. See [Display](https://getbootstrap.com/docs/4.1/utilities/display/) in bootstrap documentation.

#### Please, lets do this for all other occurrences. :blush:

File: css/style.css [hunk 5]
```diff
@@ -0,0 +1,157 @@
...
+}
+
+.topper {
+  height: 50px;
+  bottom: 0;
+  padding: 13px;
+}
+
+.wrap {
+  display: block;
```
Comment (bolah2009) [inline css/style.css]: Consider using a `Bootstrap` class name for this selector element and in all other occurrences instead.

`d-block` Bootstrap class name can be used instead. See [Display](https://getbootstrap.com/docs/4.1/utilities/display/) in bootstrap documentation.

#### Please, lets do this for all other occurrences. :blush:

File: css/style.css [hunk 6]
```diff
@@ -0,0 +1,157 @@
...
+  position: relative;
```
Comment (bolah2009) [inline css/style.css]: Consider using a `Bootstrap` class name for this selector element and in all other occurrences instead.

`position-relative` Bootstrap class name can be used instead. See [Position](https://getbootstrap.com/docs/4.1/utilities/position/) in bootstrap documentation.

#### Please, lets do this for all other occurrences. :blush:

**Comment by Oitur** [comment]
...sorry @stratospherique, but in my microverse dashboard, I still see "changes required". Shoud I submit another review request?

**Comment by stratospherique** [comment]
> ...sorry @stratospherique, but in my microverse dashboard, I still see "changes required". Shoud I submit another review request?

Sometimes it takes time. Also you should merge this Pull request because it's approved :100:

**Pull request merged**